src/module/reservation.js

```diff
@@ -0,0 +1,142 @@
+const showReservations = (e) => {
+  const popup = document.createElement('div');
+  document.body.appendChild(popup);
+  popup.classList.add('popUp', 'popUpContainer');
+  popup.style.height = '100%';
+  popup.style.width = '100%';
+  popup.style.position = 'fixed';
+  popup.style.top = '0';
+  popup.style.left = '0';
+  popup.style.backgroundColor = '#d3d3d3';
+  popup.style.zIndex = '1';
+  popup.style.display = 'grid';
+  popup.style.borderRadius = '10px';
+  popup.style.boxShadow = '0px 0px 10px rgba(0,0,0,0.5)';
+  popup.style.overflowY = 'auto';
```

Comment:
Hey, @shuja-shah Would Please mind removing the styling to set the DOM free and clean by putting them in the appropriate CSS file? Thanks